Edited the 12.0-a.7 changelog for better readability. (#29735)
Automattic/jetpack: projects/plugins/jetpack/CHANGELOG.md

## 12.0-a.7 - 2023-03-20
### Enhancements

> - Blocks: the cookie consent is now available to general audience. [#29462]

Reviewer: ```suggestion
- Blocks: the cookie consent block is now available to general audience. [#29462]
```

- Blocks: the VideoPress block is now available to general audience. [#29459]

> - Blocks: the writing prompts is now available to general audiences. [#29433]

Reviewer: ```suggestion
- Blocks: the writing prompts block is now available to general audience. [#29433]
```

- Notifications: remove all library dependencies for improved performance and lower footprint. [#29510]

> - Paid newsletter block: Improve logic for showing the UI based on site configuration. [#29262]

Reviewer: ```suggestion
- Paid newsletter block: improve logic for showing the UI based on site configuration. [#29262]
```

### Improved compatibility
- Blocks: ensure the Pinterest block does not trigger errors when using WordPress 6.2. [#29505]
- Connection: improve the connection flow that happens after plugin activation. [#29526]

### Bug fixes
- Blocks: avoid warnings in the block editor because of invalid categories for some blocks. [#29513]

> - Blocks: Fix the VideoPress preload behavior when "none" is chosen. [#29563]

Reviewer: ```suggestion
- Blocks: fix the VideoPress preload behavior when "none" is chosen. [#29563]
```

- Forms: avoid PHP notices when using a form with a dropdown field. [#29512]

### Other changes <!-- Non-user-facing changes go here. This section will not be copied to readme.txt. -->
- Adds the `is_wpcom_staging_site` attribute in a few more contexts. [#29453]
- Blocks: update deprecated block editor selector. [#29420]
- Blocks: update deprecated component prop. [#29463]

> - Firewall: Added updateWafSettings method. [#29299]

Reviewer: ```suggestion
- Firewall: add updateWafSettings method. [#29299]
```

> - Jetpack Forms: Add tracking of Google Sheets exports. [#29225]

Reviewer: ```suggestion
- Jetpack Forms: add tracking of Google Sheets exports. [#29225]
```

> - Paid newsletters: Add the launchpad source to memberships status endpoint. [#29468]

Reviewer: ```suggestion
- Paid newsletters: add the launchpad source to memberships status endpoint. [#29468]
```
